apps/golden-sample-app-e2e/data/transactions-detail.data.ts

```diff
@@ -0,0 +1,17 @@
+
+export const transactionDetailMocksData= {
```

Comment:
Not sure if that's the right place, but IMHO we need to make it more appealing that the mocks from the *-data.ts are only to be used for the tests running within the SDLC. WDYT?